### ISSUE TITLE
video with privacy "password"

### ISSUE THREAD

**Issue by TaktakTaktouk** [vimeo/vimeo.php#195]
I need to access a video protected by a password then I want to set it again to the same privacy. How can I do that? Is it possible to read the password and set it again?

**Comment by tommypenner** [comment]
The password is only used with the Vimeo Player; it's not exposed using the API. However, you can update the password by editing the video: https://developer.vimeo.com/api/reference/videos#edit_video

**Comment by TaktakTaktouk** [comment]
I want to change the privacy to "anybody" for a few minutes then set it again to "password" by using the same password. how I do that if I can't read the password with API?

**Comment by tommypenner** [comment]
Your authentication token allows you to edit the video's settings -- you don't need to know the password.

**Comment by TaktakTaktouk** [comment]
Yes, I know! but I need to set the video to "public" then set it again to "password" again.
My question is how to get the password of a video protected by a password?

**Comment by erunion** [comment]
@TaktakTaktouk We currently don't surface video passwords in the API, but after looking at our logic surrounding why that is, we're going to be changing that. I unfortunately don't have an ETA on that, but it will happen soon.

**Comment by TaktakTaktouk** [comment]
Thank you!
It will be awesome if you show the password of a protected video with the API.
Thanks for keeping me informed when you changing that.

**Comment by wmalexander** [comment]
Any news on this feature addition? It looks like password is still not available in the API

**Comment by teo1978** [comment]
I have an even more basic issue:

When I request the api endpoint `/video/12345` (where 12345 is the video ID), I get an error: 

```
[invalid_parameters] => Array
                (
                    [0] => Array
                        (
                            [field] => password
                            [error_code] => 2223
                            [error] => Whoops! Please enter a password.
                            [developer_message] => Password not provided for a password protected video.
                        )

                )
```

I get this even if I use an authenticated ("as me") access tocken with Private scope.

Is there a workaround?
